### DISCUSSION TITLE
Role Based Access Control

### DISCUSSION THREAD

**Discussion by KeyTerminal** [pelican-dev/panel#50] [Ideas]
Hi,

I'd love to see Role Based Access Control implemented as it is a standard security feature, especially
coming from a company perspective there needs to more control over admin users rather than just
they are root admin or they aren't.

The system is already there for the API key when you specifiy what permissions it has
so if that can moved across for users it would enable myself and others to manage permissions and
users through an external Identity provider making it easier to manage.

I also see #37 has asked for something similar.

**Comment by alexevladgabriel** [comment]
Closed due to the existence of https://github.com/pelican-dev/panel/discussions/37.
